Sources/OpenTelemetry/Resource/ProcessResourceDetector.swift

```diff
@@ -34,8 +34,6 @@ extension OTel {
             if #available(macOS 10.12, *) {
                 attributes["process.owner"] = ProcessInfo.processInfo.userName
             }
-            #else
```

Comment:
The `#else` here wan't referring to other Apple platforms but Linux, so we'd still need to keep it in place. Perhaps, if we replace the `#else` with `#elseif os(Linux)` it should continue to work on Linux, but now also compile on iOS/iPadOS. What do you think?

Reply:
Awesome, the tests now pass on Linux as well 👍